src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineRewriter.vb

```diff
@@ -375,6 +375,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Sub
 
+        Friend Function EnsureSpecialMember(member As SpecialMember, bag As DiagnosticBag) As Symbol
```

Comment:
@AlekseyTs I'll do that, but I won't be able to convert all the checks to DiagnosticBag just yet.

See the first line:
```VB
        Friend Overrides Function EnsureAllSymbolsAndSignature() As Boolean
            Dim hasErrors As Boolean = MyBase.EnsureAllSymbolsAndSignature

            If Me.Method.IsSub OrElse Me._elementType.IsErrorType Then
                hasErrors = True
            End If

            Dim bag = DiagnosticBag.GetInstance()
```